ansible_collections/arista/avd/molecule/eos_designs_unit_tests_v4.0/inventory/group_vars/DC1_FABRIC.yml

```diff
@@ -308,3 +308,10 @@ platform_speed_groups:
 boot:
   secret:
     key: a153de6290ff1409257ade45f
+
+hardware_counters:
```

Comment:
Please move this test to `eos_designs_unit_tests` and create a single hostvars file to create a minimal config to only test this.
Look at some of the other hostvars examples in there.

Reply:
Moved to `eos_designs_unit_tests`